app/src/main/java/org/mozilla/tv/firefox/tabs/TabsAdapter.kt

```diff
@@ -0,0 +1,106 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.tabs
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import mozilla.components.browser.session.Session
+import mozilla.components.concept.tabstray.TabsTray
+import mozilla.components.support.base.observer.Observable
+import mozilla.components.support.base.observer.ObserverRegistry
+import org.mozilla.tv.firefox.R
+
+const val ADD_VIEW_TYPE = 300
+const val SESSION_VIEW_TYPE = 200
```

Comment:
Nit: would you be alright with changing these to 0 and 1, or something similar?  With the current numbers, my mind immediately jumps to animations or something else timing related.

Although I do know that's silly, given the names.